custom_components/tahoma/__init__.py

```diff
@@ -56,7 +56,7 @@ async def async_setup(hass: HomeAssistant, config: dict):
         configuration.get(CONF_USERNAME) in entry.data.get(CONF_USERNAME)
         for entry in hass.config_entries.async_entries(DOMAIN)
     ):
-        return True
+        return False
```

Comment:
This was a mistake, we are not setting up the entity here, we cancel it.

Reply:
If you return False I think it means an error occur during the setup.